Issue 3410 styling updates

shane-tomlinson opened the pull request:
@jedp - tag, you're it!
- Add a Persona watermark next to email addresses in the email picker
- Update the "submit" button style
- Give the buttons in "Is this your computer" more space
- Shrink the TOS/PP text by 1 pt.
- Update the :focus highlight on an input element
- Fix blue cancel button on Android 2.2/2.3 native browser.

This can be seen on all-together-now.

Commit:
Put a nice Persona icon next to an email address if there is only one.

Commit:
Add a little Persona icon next to the email addresses in the email picker when in mobile mode.

partial fix for issue #3410

Commit:
Update styles as per Michael Maslaney and Maureen Hanratty's suggestions.

Commit:
Update the persona watermarks with high res versions.

rename email_input_blurred_bg=>persona_watermark
rename email_input_blurred_selected_bg=>persona_watermark_dark

Commit:
Remove the duplicate .buttonrow a, it was messing with Android 2.2/Android 2.3 browsers and causing the cancel button to be blue.
fixes #3446

jedp commented:
![2013-05-21-11-55-29](https://f.cloud.github.com/assets/532953/550952/7db09990-c31e-11e2-8d1c-662c125c354d.png)

It looks like there's a resize artifact on the right-hand side of those images.

I've tried again since Michael sent the high-res assets, but I'm still seeing the artifacts.

Apart from that, the flow and function seems perfect.

shane-tomlinson commented:
Wah wah, I'll have a look at what that artifact is. Thanks for the thorough review @jedp - I didn't notice that in my own testing until you pointed it out!

Commit:
Add an active/focus state on mobile buttons.

Commit:
Use correct sized images for the persona watermark in the email picker

This is to get rid of the anti-aliasing rendering artifact that displays on scaled images in FirefoxOS.

jedp commented:
The artifact on the right-hand side is said to be a rendering bug on gaia.  We'll follow that up in a bugzilla bug and link from here
